### PR TITLE
[Fetch] Switch branch of jsk_pr2eus to develop/fetch

### PR DESCRIPTION
**Pull request by tkmtnt7000**
To reflect https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/495 until that is merged to master branch.

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic [hunk 1]
```diff
@@ -59,12 +59,18 @@
 - git:
     local-name: jsk-ros-pkg/jsk_demos
     uri: https://github.com/jsk-ros-pkg/jsk_demos.git
-    version: master
+    version: develop/fetch
```
Comment (sktometometo) [inline jsk_fetch_robot/jsk_fetch.rosinstall.melodic:62]: Why did you also change the branch of jsk_demo?

Comment (tkmtnt7000) [reply jsk_fetch_robot/jsk_fetch.rosinstall.melodic]: Sorry, I made a mistake...

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic [hunk 2]
```diff
@@ -60,11 +60,17 @@
     local-name: jsk-ros-pkg/jsk_demos
     uri: https://github.com/jsk-ros-pkg/jsk_demos.git
     version: master
+# we need to use the development branch until PR below are merged.
+#  - https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/495
 # we need to use the development branch until the next release.
+# - git:
+#     local-name: jsk-ros-pkg/jsk_pr2eus
+#     uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+#     version: develop/fetch
```
Comment (sktometometo) [inline jsk_fetch_robot/jsk_fetch.rosinstall.melodic:69]: I think if you uncomment after merging the PRs, verion here should be master.

Comment (tkmtnt7000) [reply jsk_fetch_robot/jsk_fetch.rosinstall.melodic]: Thank you. I fixed.

**Review by sktometometo — APPROVED** [review]

**Comment by tkmtnt7000** [comment]
Thank you!

**Comment by k-okada** [comment]
I have merged https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/495 so you can revert this PR now
I also changed hoge.py to use insert_at instead of headers. -> https://github.com/tkmtnt7000/jsk_demos/commit/9e4b9885bac668494e8756c3e1a956ea441a7da8#diff-61e0efee72c93663b508c3bb5b33dad88a335d7e66c39aae3dc9a0a4702c20e9R288
Originally I thought this is darty code, but if we have more topics without correct header stamp, may be this is good strategy.

**Pull request merged**